src/views/Layout.vue

```diff
@@ -178,14 +178,31 @@ export default {
      */
     this.$root.$on('alert', ({ message, ...params }) => this.toast(message, params))
     this.$root.$on('reminder.show', this.showReminder)
+
+    this.$root.$on('sidebar-show', this.checkNamespaceSidebar)
```

Comment:
lets keep the event name similar to method name, so 'check-namespace-sidebar'
Since sidebar show is not really describing it correctly